estate/models/estate_property.py

```diff
@@ -0,0 +1,52 @@
+from odoo import fields, models
+from odoo.tools.date_utils import add
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate properties"
+
+    name = fields.Char("Title", required=True)
+    description = fields.Text("Description")
+    postcode = fields.Integer("Postcode")
+    date_availability = fields.Date(
+        "Available from", copy=False, default=add(fields.Date.today(), months=3)
+    )
+    expected_price = fields.Float("Expected Price", required=True)
+    selling_price = fields.Float("Selling Price", readonly=True, copy=False)
+    bedrooms = fields.Integer("Bedrooms", default=2)
+    living_area = fields.Integer("Living Area (sqm)")
+    facades = fields.Integer("Facades")
+    garage = fields.Boolean("Garage")
+    garden = fields.Boolean("Garden")
+    garden_area = fields.Integer("Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ("north", "North"),
```

Comment:
This example is better to explain the difference: first one is only used internally to check the selection choice, while the second is displayed to the customer, so
```suggestion
            ('north', "North"),
```